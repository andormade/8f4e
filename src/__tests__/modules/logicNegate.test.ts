import moduleTypeGenerator from '../../modules/logicNegate';

it('generates correct module type descriptor', () => {
	expect(moduleTypeGenerator()).toMatchSnapshot();
});
