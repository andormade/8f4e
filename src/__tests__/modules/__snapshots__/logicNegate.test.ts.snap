// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generates correct module type descriptor 1`] = `
Object {
  "buttons": Array [],
  "category": "Logic",
  "engine": Object {
    "config": Object {},
    "name": "logicNegate",
  },
  "height": 126,
  "initialState": Object {},
  "inputs": Array [
    Object {
      "height": 13,
      "id": "in",
      "width": 12,
      "x": 12,
      "y": 42,
    },
  ],
  "lines": Array [
    Object {
      "height": 1,
      "width": 108,
      "x": 6,
      "y": 7,
    },
    Object {
      "height": 112,
      "width": 1,
      "x": 114,
      "y": 7,
    },
    Object {
      "height": 1,
      "width": 108,
      "x": 6,
      "y": 119,
    },
    Object {
      "height": 112,
      "width": 1,
      "x": 6,
      "y": 7,
    },
  ],
  "name": "Logic Negate",
  "outputs": Array [
    Object {
      "height": 13,
      "id": "out",
      "width": 12,
      "x": 96,
      "y": 42,
    },
  ],
  "sliders": Array [],
  "steppers": Array [],
  "width": 120,
}
`;
